src/pythium/sklimming/config.py

```diff
@@ -87,20 +87,15 @@ def validate_branches(branches, sample_name, tree):
     for branch in branches:
```

Comment:
Exit neatly if recurring branch dependency error found

Reply:
done. RecursionError now explained to user and culprit branches printed